spec/support/simplecov.rb

```diff
@@ -0,0 +1,7 @@
+require "coveralls"
+require "simplecov"
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+  SimpleCov::Formatter::HTMLFormatter,
+  Coveralls::SimpleCov::Formatter
+]
+SimpleCov.start("rails")
```

Comment:
@bhaibel - I wasn't sure if we were intentionally not including the rails stuff or if we weren't. If we are intentionally not including rails I can ditch this

Reply:
When you say "the Rails stuff" what do you mean? I have mixed feels about including Rails config stuff in coverage, because on the one hand I want people to Think about how much stuff they sling in there and on the other hand I think that it can serve as a broken window. I think that controllers should be included in coverage.

Reply:
To clarify, I mean the controllers/jobs/channels/mailers/etc, as shown here: https://coveralls.io/builds/14790097


Previously, coverage was only being reported for files which were explicitly tested.